src/test/ui/parser/qpath-in-type-args.rs

```diff
@@ -0,0 +1,11 @@
+trait Tr {
+    type TrSubtype;
```

Comment:
```suggestion
    type TrAssoc;
```

Why is this called "Subtype" (... what is the relation to subtyping?)

Reply:
I took the repro case from the original report verbatim without paying attention to what it said 😅